waku/v2/node/wakunode2_test.go

```diff
@@ -318,3 +323,218 @@ func TestDecoupledStoreFromRelay(t *testing.T) {
 	require.Len(t, result.Messages, 1)
 	require.Equal(t, msg.Timestamp, result.Messages[0].Timestamp)
 }
+
+func TestStaticShardingMultipleTopics(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	nodeNativeClusterID := uint16(20)
+	shardingClusterID := uint16(21)
+
+	// Node1 with Relay
+	hostAddr1, err := net.ResolveTCPAddr("tcp", "0.0.0.0:0")
+	require.NoError(t, err)
+	wakuNode1, err := New(
+		WithHostAddress(hostAddr1),
+		WithWakuRelay(),
+		WithClusterID(nodeNativeClusterID),
+	)
+	require.NoError(t, err)
+	err = wakuNode1.Start(ctx)
+	require.NoError(t, err)
+	defer wakuNode1.Stop()
+
+	pubSubTopic1 := protocol.NewStaticShardingPubsubTopic(shardingClusterID, uint16(0))
+	pubSubTopic1Str := pubSubTopic1.String()
+	contentTopic1 := "/test/2/my-app/sharded"
+
+	pubSubTopic2 := protocol.NewStaticShardingPubsubTopic(shardingClusterID, uint16(10))
+	pubSubTopic2Str := pubSubTopic2.String()
+	contentTopic2 := "/test/3/my-app/sharded"
+
+	require.Equal(t, nodeNativeClusterID, wakuNode1.ClusterID())
+
+	r := wakuNode1.Relay()
+
+	subs1, err := r.Subscribe(ctx, protocol.NewContentFilter(pubSubTopic1Str, contentTopic1))
```

Comment:
Described above. If this usage is not intended by devs, it would be better to prevent it in the code. 

Is the cluster ID an administrative concept ? More like a sticky label of a node for human operator?  ENR is hard to remember, otherwise ENR+pubsub topic does the real work with message routing. Am I getting it?

Reply:
Gave a brief explanation https://github.com/waku-org/go-waku/pull/1060#discussion_r1587048424

Reply:
Also, you can raise an issue so that we don't forget about this.

Although not sure if we want to implement in go-waku as it is being sunset.

Reply:
Let's keep it in this PR only. I don't want to generate additional work, if that is what sunsetting means.

Reply:
My bad, no issue with this comment. 

I was talking about https://github.com/waku-org/go-waku/pull/1060#discussion_r1587172611.